contracts/ethereum/src/core/mmr/mocks/MMRWrapper.sol

```diff
@@ -0,0 +1,41 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.17;
+
+import "../MMR.sol";
+
+contract MMRWrapper is MMR {
+
+  bytes public rootHash;
+  bool public verified;
+
+  constructor() {
+    blake2bInstance = Blake2b.init(hex"", 64);
+    scaleCodec = new ScaleCodec();
```

Comment:
Isn't passing the address of an existing contract better than deploying a new one?